test/accessor.test.ts

```diff
@@ -0,0 +1,448 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { test } from 'node:test';
+import assert from 'node:assert';
+import { stub, SinonStub } from 'sinon';
```

Comment:
Please use `mock` from `node:test` instead

Reply:
Filed an issue on the repo to migrate from sinon: https://github.com/google-github-actions/analyze-code-security-scc/issues/15

Reply:
I know your launch is soon, but I'd prefer doing it in this PR. It will also resolve the unnecessary dependency injection of a custom httpClient, since you can mock the prototype directly with mock from node:test.